### PR TITLE
added raw tables.

### PR DESCRIPTION
**Pull request by heidariank**
Test plan:
For the moment, I would just like someone to look over these quickly to double check since this was a lot of copy-paste.

### CODE REVIEW

File: lib/political_leaning/raw_data/raw_individual_contributor.ex [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    field(:FILE_NUM, :integer)
+    field(:MEMO_CD, :string)
+    field(:MEMO_TEXT, :string)
+    field(:SUB_ID, :integer)
+    timestamps()
+  end
+
+  def changeset(raw_individual_contributor, params) do
+    raw_individual_contributor
+    |> cast(params, @fields)
```
Comment (matelau) [inline lib/political_leaning/raw_data/raw_individual_contributor.ex]: should we avoid the single pipe here?

Comment (heidariank) [reply lib/political_leaning/raw_data/raw_individual_contributor.ex]: Yes, I meant to change that. Thanks.

**Comment by heidariank** [comment]
> I added one comment. I think adding tests will be nice we want to create good patterns for those learning. Also, what do you think about downcasing and normalizing the fields? ie CAND_ID -> candidate_id?

I do prefer lowercase field names but I kind of like keeping the schema as close to the original as possible. What benefit do you see in downcasing the fields?

**Comment by matelau** [comment]
> > I added one comment. I think adding tests will be nice we want to create good patterns for those learning. Also, what do you think about downcasing and normalizing the fields? ie CAND_ID -> candidate_id?
> 
> I do prefer lowercase field names but I kind of like keeping the schema as close to the original as possible. What benefit do you see in downcasing the fields?

I just thought it would be more explicit! I do also see the value in preserving those fields as is from the original data and converting them to what we need in our table.

File: lib/political_leaning/raw_data/raw_candidate_committee_linkage.ex [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    :CAND_ID,
+    :CAND_ELECTION_YR,
+    :FEC_ELECTION_YR,
+    :CMTE_ID,
+    :CMTE_TP,
+    :CMTE_DSGN,
+    :LINKAGE_ID
+  ]
+  
+  schema "raw_committee_master" do
```
Comment (matelau) [inline lib/political_leaning/raw_data/raw_candidate_committee_linkage.ex]: should these two match RawCandidateCommitteeLinkage vs `raw_committee_master`?

File: lib/political_leaning/raw_data/raw_committee_master.ex [hunk 3]
```diff
@@ -0,0 +1,49 @@
+defmodule PoliticalLeaning.RawCommitteeMaster do
+  @moduledoc """
+  Copies data from for FEC committee master table
```
Comment (matelau) [inline lib/political_leaning/raw_data/raw_committee_master.ex]: can we add the links to include the docs for the source https://www.fec.gov/campaign-finance-data/committee-master-file-description/

**Pull request merged**